src/ast/passes/deprecated.cpp

```diff
@@ -47,7 +49,9 @@ static void check(const std::vector<DeprecatedName> &list,
 
     if (item.deleted) {
       auto &err = node.addError();
-      err << item.old_name << " is deprecated and has no effect.";
+      err << item.old_name << " is deleted. " << "This was deprecated in "
+          << item.version_deprecated << " and was last available in "
+          << item.version_last_available << ".";
```

Comment:
Missed this, but we should still include `new_name` as well as a hint here, e.g.

```
err.addHint() << "Use " << item.new_name << " instead.";
```

(I don't know if we'll be able to have that for every case, but at least it works in this case.)

Reply:
Ack, fine.